cmd/tailscale/cli/network-lock.go

```diff
@@ -377,7 +394,18 @@ func runNetworkLockDisable(ctx context.Context, args []string) error {
 var nlLocalDisableCmd = &ffcli.Command{
 	Name:       "local-disable",
 	ShortUsage: "local-disable",
-	ShortHelp:  "Disables the currently-active tailnet lock for this node",
+	ShortHelp:  "Disables tailnet lock (currently enabled) for this node only",
```

Comment:
WRT `(currently enabled)` - this command help will always be visible regardless of whether tailnet lock is enabled or not. Do we still want to have that text?

Reply:
Ah, no, I didn't get that. Removing.